app/controllers/notices-setup.controller.js

```diff
@@ -43,9 +43,9 @@ const SubmitLicenceService = require('../services/notices/setup/submit-licence.s
 const SubmitNoticeTypeService = require('../services/notices/setup/submit-notice-type.service.js')
 const SubmitRecipientNameService = require('../services/notices/setup/submit-recipient-name.service.js')
 const SubmitRemoveLicencesService = require('../services/notices/setup/submit-remove-licences.service.js')
-const SubmitReturnFormsService = require('../services/notices/setup/submit-return-forms.service.js')
 const SubmitReturnsPeriodService = require('../services/notices/setup/returns-period/submit-returns-period.service.js')
 const SubmitSelectRecipientsService = require('../services/notices/setup/submit-select-recipients.service.js')
+const submitPaperReturnService = require('../services/notices/setup/submit-paper-return.service.js')
```

Comment:
NIT. I also think it needs to be bumped up in the order now.


```suggestion
const SubmitPaperReturnService = require('../services/notices/setup/submit-paper-return.service.js')
```